### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributing to GLPI
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+Please note current repository is about GLPI core only. All related plugins are the responsibility of their respective owners and all requests must be done on their own systems.
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `Please note current repository is about GLPI core only.` -> `Please note that the current repository is containing GLPI core only.` ?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+**Write only in english!**
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `Please write only ...` ?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+If developpers are not able to reproduce the issue, it may not be fixed.
+
+Note that issues are handled on a best-effort basis. If you need a quick fix or any guarantee, take a look at [profesionnal services](http://services.glpi-network.com/) or [partners](https://glpi-project.org/partners/).
+
+## Features
+
+If you want to work on a new feature, open a ticket using the feature template, so we can discuss on it.
+
+If you're just asking for something to be added without doing it yourself, please condier adding (or upvoting) an enhancement on the [suggestion website](https//suggest.glpi-project.org).
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `condier` -> `consider`

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Support request
+
+You have a question on GLPI usage, or want to know what are the capabilities.
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `?` at the end of sentence ?

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Current repository is not the right place for that. You may try to aks your question on [the forums](https://forum.glpi-project.org) on the mailing lists, on telegram channels, ...
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `,` between forums and mailing lists.

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (trasher) [reply .github/CONTRIBUTING.md]: Depends if you ask a question or just tell something :D Guess both are quite the same, this do not change anything.

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Contributing to GLPI
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+Please note current repository is about GLPI core only. All related plugins are the responsibility of their respective owners and all requests must be done on their own systems.
+
+**Plase write only in english!**
```
Comment (cedric-anne) [inline .github/CONTRIBUTING.md]: `Plase` -> `Please`

**Pull request merged**